TwitchChannelPointsMiner/logger.py

```diff
@@ -85,10 +93,11 @@ def configure_loggers(username, settings):
         )
         file_handler = logging.FileHandler(logs_file, "w", "utf-8")
         file_handler.setFormatter(
-            EmojiFormatter(
+            GlobalFormatter(
```

Comment:
This is just a personal curiosity. 
Have you tested if the file logger works? I'm just scared to have strange "string/chars" in the file logs :thinking:

Reply:
I didn't really, but you're actually right. Here's an example:
21/02/21 18:33:33 - INFO - TwitchChannelPointsMiner.classes.WebSocketsPool - [on_message]: [31m[31mEventPrediction: mount (6.77k points) - ele passa do round 100? - Result: LOSE, Gained: -1.01k[0m[0m